app/models/requests/request.rb

```diff
@@ -132,7 +132,7 @@ def has_pageable?
     def has_loanable_copy?
       copy_available = []
       requestable_unrouted.each do |request|
-        if request.charged? || (request.aeon? || !request.circulates?) #|| request.enumerated?) 
+        if request.charged? || (request.aeon? || !request.circulates? || request.enumerated?)
```

Comment:
I though we were handling this with the borrow_direct_eligible? call since that is the only edge case that requires special handling.